internal/exec/stages/files/passwd.go

```diff
@@ -16,10 +16,29 @@ package files
 
 import (
 	"fmt"
+	"path/filepath"
 
 	"github.com/coreos/ignition/v2/config/v3_1_experimental/types"
 )
 
+func (s *stage) deglobList(globs ...string) ([]string, error) {
```

Comment:
If I'm reading it correctly, this first expands a list of globs on top of `s.DestDir`, then truncates each entry back to `/`, correct?

`expandGlobList` maybe?